inst/paper.md

```diff
@@ -123,6 +123,13 @@ like the Workflow Template and Toolkit System (WATTS) [@Romano:2022].
 This though requires the user to create a template from the set of problems they plan to model.
 This would be well suited for a sensitivity study where many very similar simulations are run,
 but not for making large edits to an input file or making a single problem from scratch.
+PyNE: the Nuclear Engineering Toolkit offers some similar capabilities for input generation.
+
+PyNE can create MCNP input files for specific features and extract some data from MCNP output files.
+However, its full capabilities extend far beyond interfacing with MCNP.
+PyNE can simplify material creations, analyses of cross section data, 
+transmutations of complex systems, and interfacing with other common nuclear engineering software and data formats [@scopatz:2012].
```

Comment:
References appear to be case-sensitive in markdown. Bibliography uses capitalized `@Scopatz`.